src/components/submission/decrypt.vue

```diff
@@ -74,14 +83,13 @@ except according to the terms contained in the LICENSE file.
           </div>
           <div class="submission-download-action"
             :class="{ disabled: noRepeat }">
-            <span class="submission-download-action-label"
-              :title="noRepeat ? $t('noRepeat') : null">
+            <span class="submission-download-action-label">
```

Comment:
I tried using `v-tooltip` here, but that gives the label a separate tooltip (separately positioned) from the link below it. That seemed a little funny to me, but I think it's fine for just the link to get the tooltip.